File: unix/boot/spp/rpp/rppfor/swvar.f
      subroutine swvar (lab)
      integer lab, i, labnum, ndigi0
      call outch (115)
      call outch (119)
      labnum = lab
      ndigi0=0
23000 if (.not.(labnum .gt. 0))goto 23002
      ndigi0 = ndigi0 + 1
23001 labnum=labnum/10
      goto 23000
23002 continue
      i=3
23003 if (.not.(i .le. 6 - ndigi0))goto 23005
      call outch (48)
23004 i=i+1
      goto 23003
23005 continue
      call outnum (lab)
      return
      end
c     ndigi0  ndigits

File: unix/boot/spp/rpp/rpprat/swvar.r
#-h-  swvar			  157  local   12/01/80  15:55:08
# swvar - output switch variable SWnnnn, where nnnn = lab
# (modified aug82 dct to permit declaration of switch variable)

   include  defs

   subroutine swvar (lab)
   integer lab, i, labnum, ndigits

   call outch (LETS)
   call outch (LETW)

   labnum = lab
   for (ndigits=0;  labnum > 0;  labnum=labnum/10)
    	ndigits = ndigits + 1
   for (i=3;  i <= 6 - ndigits;  i=i+1)
	call outch (DIG0)
   call outnum (lab)
   return
   end
